back/test/endpoint_tests/test_statistics.py

```diff
@@ -395,10 +399,12 @@ def test_authorization(read_only_client, mocker):
     ]:
         assert_forbidden_request(read_only_client, query)
 
+    # Test case 11.1.5
     # Base 5 does not exist
     query = "query { createdBoxes(baseId: 5) { facts { productId } } }"
     assert_forbidden_request(read_only_client, query)
```

Comment:
@pylipp shouldn't this be an invalid data response rather than a forbidden response?

Reply:
@aerinsol my thoughts were: if a forbidden response is returned, it isnt even revealed that the base does not exists which I find more secure.
Also for checking invalid data a DB query is required but for authz check it's not.

Reply:
I see, it's part of the series of tests for unauthorized access to show that now more information is shown than needed. Thanks for clarifying.